ironfish/src/network/peers/peerAddressManager.test.ts

```diff
@@ -0,0 +1,21 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+jest.mock('ws')
+
+import { mockHostsStore } from '../testUtilities'
+import { PeerAddressManager } from './peerAddressManager'
+
+jest.useFakeTimers()
+
+describe('PeerAddressManager', () => {
+  it('constructor load hosts from HostsStore', () => {
+    const peerAddressManager = new PeerAddressManager(mockHostsStore())
+    expect(peerAddressManager.priorConnectedPeerAddresses).toMatchObject([
```

Comment:
I suppose you could add a test for `possiblePeerAddresses` as well

Reply:
👍🏽